idb/postgres/internal/migrations/convert_account_data/m_test.go

```diff
@@ -26,8 +26,7 @@ func makeAddress(i int) basics.Address {
 }
 
 func insertAccount(t *testing.T, db *pgxpool.Pool, address basics.Address, trimmedAccountData *basics.AccountData) {
-	query := `INSERT INTO account (addr, microalgos, rewardsbase, rewards_total, deleted,
-		created_at, account_data) VALUES ($1, 0, 0, 0, false, 0, $2)`
+	query := `INSERT INTO account (addr, microalgos, rewardsbase, rewards_total, deleted, account_data) VALUES ($1, 0, 0, 0, false, $2)`
```

Comment:
why removing `created_at`?

Reply:
Indexer stores the round at which a particular row was inserted. I feel like this unwanted data and removing this would decrease the size of our indexer.